tests/Unit/Utilities/Test_TaggedTuple.cpp

```diff
@@ -61,7 +61,7 @@ double test_function(const int first_arg, const double second_arg) {
 // [expand_tuple_example_function]
 }  // namespace
 
-SPECTRE_TEST_CASE("Unit.Utilities.TaggedTuple", "[Utilities][Unit]") {
+void test_general() {
```

Comment:
Move into anonymous namespace.

Reply:
Thanks for catching this! Squashed and pushed :)